src/components/App/App.js

```diff
@@ -1,17 +1,46 @@
 import React from 'react';
+import './App.css';
 
 const App = props => {
+	props.loadStoreList();
 	return (
-		<div>
-			<h1>Paying Later?</h1>
+		<div id="homepage">
+			<h2 id="heading">
+				<svg
```

Comment:
I think the designs have changed a little so that the halo is only used on the logo, not headings. Could we remove this for now?